### ISSUE TITLE
Few tweaks to mimic the bear theme

### ISSUE THREAD

**Issue by dtomlinson91** [apg-dev/trilium-theme-bear#1]
Amazing work on the theme - it really makes trilium a delight to use.

You said suggestions are welcomed - I tweaked it slightly to make it look more like bear (which I've migrated from)

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/53234158/73619608-0bddac80-4626-11ea-8f0b-65743eb36eaf.png">

I extended the dark sidebar, added a bit of grey to the buttons, added a thin gutter bar on the right, and changed the default `<a>` colour to be more of a red link in bear.

If you're interestred I can share the changes? Feel free to close this off if not just wanted to share it since you've done a great job already.

**Comment by apg-dev** [comment]
I love the work you did - looks great! Yes, please share. I'm partial to the off-white background (#f9f9f9) rather than the stark white. 

I, too, migrated from Bear. I really liked Bear's aesthetics - it was joy taking notes there. But the organization just didn't work (for me) for creating a full-blown KMS. I really appreciate Trilium and where it's going.

**Comment by dtomlinson91** [comment]
Hmm you're right I don't know why it's showing white. I haven't overwritten it anywhere (I think). I've added the `#f9f9f9` to the bottom of my `.css` theme to force it, are you using a custom `styles.css` in trilium perhaps?

https://gist.github.com/dtomlinson91/6af647713f0b119374ac35ef1843ce4f

Feel free to add bits of this to your theme if you want. I prefer a smaller font size so I overwrite the size where you specified it. I'm no css expert, I've no doubt done something a bit inelgant somewhere. Some of the `!important;` rules are probably not needed, I was doing it on the fly in the console and just forced it out of habit. I also probably didn't catch everything, I bet there's a few things I missed.

Bear was (is) a brilliant piece of software and well worth the money. Since discovering Trilium and the way it handles its trees with the really robust tagging and relationship attributes I don't think I can go back to bear.

On a side note, where did you find the back/forward buttons in top-left of your screenshot. They're not there for me and I can't find an option in the settings.

**Comment by apg-dev** [comment]
I appended your CSS to the end of mine and it looks great. The only change I needed to make was due to the link-color variable not changing all links to the reddish color (only some), so I needed to change my "a" tag to color: #A02F2F !important. Also I don't have Soleil fonts - need to get it - so I had to comment those lines out for now. But the background color is indeed #f9f9f9. I wonder if it's just the screenshot you created that looks white to me. Your code (and my machine) are definitely showing the f9 background color, which is nice.

Good call on the font sizes. I think I cranked my Bear settings pretty far up and I just modeled what I had in there. But the smaller font does look better.

The back/forward buttons are part of the desktop version of the app. They don't show up in the browser version - presumably since the browser already has that functionality. Are you hosting Trilium on your own server?

I, too, like Trilium for the reasons you point out. It still has some work to do in the full link mapping space - and I know the developer is working on enhancements in that area. Here is a brief thread between us.

---

Me: If using tag attributes (e.g. @tag =productivity,  @tag=note-taking), it would be helpful for a Link Tag Map widget to show all notes that contain the same tags as the current note → ideally with each of them hyperlinkable. Otherwise, only other option is to search for “@tag=tagname”. But that won’t quickly show all notes with all tags that are within the current note. You have to search one tag at a time and mentally build your own link tag map.

Zadam: I like that idea. In general I plan to work on making related notes more discoverable - so far this is done through link map (relations) and “similar notes” (looks for similar note titles), in the future I plan to add more advanced content analysis and also this label based context (which would be also pretty simple).

---

Meantime, while that is being developed, in addition to Trilium I am also using Roam (roamresearch.com). It’s a radically different paradigm for knowledge management (e.g. there is no concept of folders or trees), but it presents some really interesting use cases - particularly for a zettelkasten-like framework.

**Comment by dtomlinson91** [comment]
Yeah I'm hosting it, the mac desktop version doesn't play nice and I can understand the dev not wanting to bother if he doesn't own one himself (I enjoy my mac but I can understand how they're a massive pain in the arse).

The suggestion for tags like that is actually a great idea - the similar notes I don't find quite useful but if the similarity was based on similar tags (and combinations thereof) it would be a great addition.

I'm struggling to understand what Roam is for, I've never heard of zettelkasten!

**Comment by apg-dev** [comment]
I think the tag infrastructure will be really helpful for Trilium - and I'm greatly looking forward to it. That's really the core of Roam's architecture: treating the note as the fundamental element (versus the notebook or folder or category or even tag) - and using bi-directional links to maintain connections.

I stumbled across Roam after reading a book called How to Take Smart Notes by Sönke Ahrens. The book describes how a German sociologist named Niklas Luhmann produced a mind-boggling amount of quality papers (and thoughts) across many diverse disciplines including sociology, philosophy, education, psychology, law, politics, economy, communication, and art. His "secret" - which Luhmann was never secret about - was his zettelkasten or slip-box. Basically his note taking system (framework and methodology). His slip-box not only provided a nearly unlimited supply of content and ideas, but - by design - created unanticipated, highly fortuitous connections between his notes.

I was fascinated by Ahren's description of Luhmann and began exploring relevancy for me. And in my various internet meanderings, I came across both Trilium and Roam - two early stage, powerful products that resonate with me and my workflow.
